arangod/VocBase/LogicalCollection.cpp

```diff
@@ -622,15 +621,14 @@ Result LogicalCollection::rename(std::string&& newName) {
     return TRI_ERROR_INTERNAL;
   }
 
-  auto doSync = databaseFeature.forceSyncProperties();
   std::string oldName = name();
 
   // Okay we can finally rename safely
   try {
     StorageEngine& engine =
         vocbase().server().getFeature<EngineSelectorFeature>().engine();
     name(std::move(newName));
-    engine.changeCollection(vocbase(), *this, doSync);
+    engine.changeCollection(vocbase(), *this, /*doSync*/ false);
```

Comment:
Can we get rid of this `doSync` argument as well?

Reply:
Done